diff -Naurp xca-2.4.0.orig/Makefile xca-2.4.0.new/Makefile
--- xca-2.4.0.orig/Makefile	2021-05-07 22:40:29.000000000 +0200
+++ xca-2.4.0.new/Makefile	2021-11-13 16:59:53.656274074 +0100
@@ -69,7 +69,9 @@ PRINT=:
 endif
 export PRINT
 
-ifneq ($(TOPDIR), $(BUILD))
+REALBUILD=$(shell realpath $(BUILD))
+REALTOPDIR=$(shell realpath $(TOPDIR))
+ifneq ($(REALTOPDIR), $(REALBUILD))
 do.ui: clean_topdir
 clean_topdir:
 	$(MAKE) -C $(TOPDIR) clean
