From f5910dd1ab60780b95eed16d36860f18b01bc156 Mon Sep 17 00:00:00 2001
From: Saket Dumbre <saket.dumbre@intel.com>
Date: Wed, 18 Oct 2023 15:30:43 -0700
Subject: [PATCH] Attempt 1 to fix issue #900

---
 source/components/utilities/utdebug.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/components/utilities/utdebug.c b/source/components/utilities/utdebug.c
index 82e8a62050..63739ee220 100644
--- a/source/components/utilities/utdebug.c
+++ b/source/components/utilities/utdebug.c
@@ -190,7 +190,6 @@ AcpiUtInitStackPtrTrace (
 #pragma GCC diagnostic ignored "-Wdangling-pointer="
 #endif
     AcpiGbl_EntryStackPointer = &CurrentSp;
-#pragma GCC diagnostic pop
 }
 
 
@@ -216,6 +215,7 @@ AcpiUtTrackStackPtr (
     if (&CurrentSp < AcpiGbl_LowestStackPointer)
     {
         AcpiGbl_LowestStackPointer = &CurrentSp;
+#pragma GCC diagnostic pop
     }
 
     if (AcpiGbl_NestingLevel > AcpiGbl_DeepestNesting)
