--- ./setup.py  2021-09-15 16:35:03.918459829 +0200
+++ ./setup.py  2021-09-15 16:33:57.806341852 +0200
@@ -18,7 +18,7 @@
     license="BSD",
     keywords="cryptography API NaCl libsodium",
     url="https://github.com/stef/pysodium",
-    packages=find_packages(),
+    packages=find_packages(exclude=['test*']),
     long_description=read('README.md'),
     requires=["libsodium"],
     classifiers=["Development Status :: 4 - Beta",
