From 2d188e49503c07b74ff029b4c44098849c917971 Mon Sep 17 00:00:00 2001
From: Merlijn Wajer <merlijn@wizzup.org>
Date: Fri, 24 Jan 2020 14:03:53 +0100
Subject: [PATCH] includes/usbg/usbg.h: include sysmacros.h for
 makedev/major/minor

---
 include/usbg/usbg.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/usbg/usbg.h b/include/usbg/usbg.h
index 55c52a1..b9473f0 100644
--- a/include/usbg/usbg.h
+++ b/include/usbg/usbg.h
@@ -29,6 +29,7 @@
 #include <stdbool.h>
 #include <stdio.h> /* For FILE * */
 #include <malloc.h>
+#include <sys/sysmacros.h>
 
 #include "usbg_version.h"
 
